core/ui/LayoutSwitcher.tid

```diff
@@ -5,7 +5,7 @@ caption: {{$:/language/ControlPanel/LayoutSwitcher/Caption}}
 \whitespace trim
 <$linkcatcher to="$:/layout">
 <div class="tc-chooser">
-<$list filter="[all[tiddlers+shadows]tag[$:/tags/Layout]] [[$:/core/ui/PageTemplate]] +[!is[draft]sort[name]]">
```

Comment:
The reason that we manually include `$:/core/ui/PageTemplate` is so that the default page template is shown in the chooser even if the user is using a customised `$:/core/ui/PageTemplate` that dates from before the introduction of switchable layouts, and hence lacks the tag `$:/tags/Layout`.

Was the intention just to clean things up, or was the current arrangement causing problems for you?

Reply:
That is reasonable, added back!

I was trying to save a bit of calculation...